### ISSUE TITLE
feat: Support Content v3

### ISSUE THREAD

**Issue by zernonia** [ZTL-UwU/shadcn-docs-nuxt#73]
### Describe the feature

It will be awesome to support [Content v3](https://content3.nuxt.dev/docs/getting-started/installation). 

I understand that would required major refactoring, seeing there's multiple composables were [gone](https://content3.nuxt.dev/docs/getting-started/migration#queryfind-contents), but I believe it's gonna help dev easily adopt this template as a starter for Content 3 site 😁

idea: Maybe a dedicated branch for v3? 

### Additional information

- [ ] Would you be willing to help implement this feature?

**Comment by ZTL-UwU** [comment]
Yes, I'm planning to move to v3 in the 1.0 major release along with some breaking changes to clean up the config. I'm currently working on it already, maybe I'll create a v3 branch soon.

**Comment by zernonia** [comment]
While you were there, maybe can use [shadcn-vue@next](https://next.shadcn-vue.com/) with `reka-ui` for this 1.0? 😁

**Comment by ZTL-UwU** [comment]
> While you were there, maybe can use [shadcn-vue@next](https://next.shadcn-vue.com/) with `reka-ui` for this 1.0? 😁

Definitely!
